components/common/Button/button.scss

```diff
@@ -17,35 +16,80 @@
 	}
 
 	&--primary {
-		background-color: $button-primary-background-color;
-		color: $button-text-color;
+		background-color: $blue-light;
+		color: $button-text-light;
 	}
 
 	&--secondary {
-		color: $button-secondary-text-color;
+		color: $blue-light;
 		background-color: transparent;
 	}
 
+	&--gray,
+	&--transparent {
+		color: $text-lighter;
+		font-weight: 400;
+	}
+
+	&--transparent,
+	&--red {
+		background-color: transparent;
+	}
+
+	&--red {
+		color: $red;
+		border: 0.5px solid $red;
+	}
+
+	&--greenIcon {
+		color: $white;
+		background-color: $green;
+		border: none;
+	}
```

Comment:
nie powinno być nazewnictwa powiązanego z kolorem, zazwyczaj to jest bardziej co robi dana akcja np. red -> alert, yellow -> warning, green -> success, lub jakiś default, secondary, hidden, etc.